kuma/wiki/templates/wiki/revision.html

```diff
@@ -8,7 +8,7 @@
 {% block content %}
 
   <article>
-    <h1>{{ document.title }}</h1>
+    <h1><a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ _('{title}')|fe(title=document.title) }}</a></h1>
```

Comment:
TBH, I'm not even sure how `{{ _('{title}')|fe(title=document.title) }}` is supposed to work? `document.title` is pulled from the database and already localized right? Why does it need to be passed through `_` (aka `gettext`)?

I would think simply doing the follow is all you need:
`<a href="{{ url('wiki.document', document.full_path, locale=document.locale) }}">{{ document.title }}</a>`
and that should be escaped by default.